app/Http/Controllers/Forms/AllAction.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers\Forms;
+
+use App\Eloquents\Circle;
+use App\Eloquents\Form;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Carbon\CarbonImmutable;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Gate;
+
+class AllAction extends Controller
+{
+    public function __invoke()
+    {
+        $forms = Form::IsPublic()->CloseOrder()->get();
```

Comment:
スコープメソッドも先頭の文字は小文字でお願いします！

（isPublic を public に改名する場合、 Form::public() かな）

```suggestion
        $forms = Form::isPublic()->closeOrder()->get();
```